Makefile

```diff
@@ -1,3 +1,79 @@
+# Default shell
+SHELL := /bin/sh
+
+# Global CI
+LANGUAGES := go
+# Global CI. Do not edit
+MAKEFILE := Makefile.docs
+CI_REPO_URL := https://github.com/src-d/ci.git
+SHARED_PATH ?= /etc/shared
```

Comment:
The default should not be `/etc/shared`, we should avoid putting files on random places in the developer's computer. This must be a folder inside the project. Like `$(shell pwd)/.shared` or something like that.